packages/x-data-grid-generator/src/columns/employees.columns.tsx

```diff
@@ -170,7 +170,7 @@ export const getEmployeeColumns = (): GridColDefGenerator[] => [
       if (!value || typeof value !== 'number') {
         return value;
       }
-      return `${value.toLocaleString()}$`;
+      return `$${value.toLocaleString()}`;
```

Comment:
🤔 Seems like this should be 
```suggestion
      return `${value.toLocaleString()}`;
```

Reply:
Why? This is the amount in dollars, I just moved it where it needs to be.

Reply:
Yeah... now I see it as well! 🤣😅 sry about that! Thought it is a typo 🙇

Reply:
position of the currency sign also depends on the locale
you have to use [Intl.NumberFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat) to have it right
(can be dealt with in a follow up)